### ISSUE TITLE
Per-node 'active' status ignores last_relayed — repeaters that disable ADVERT but actively relay are wrongly marked stale

### ISSUE THREAD

**Issue by MarekWo** [Kpa-clawbot/meshcore-analyzer#1605]
## Summary

Per-node status calculation (`getNodeStatus()` in `public/roles.js`) considers only `last_heard` / `last_seen` (which track ADVERT freshness) and ignores `last_relayed`. Many operators in our community **deliberately disable ADVERT** on their repeaters to reduce mesh airtime — the mesh keeps working fine because routing doesn't require periodic adverts. The result is that actively-relaying repeaters fall out of the "Active" filter, disappear from the map's "Active" view, and look offline on the Nodes page — even though they're handling thousands of packets per day.

This combines with #662 (relay activity already tracked server-side) — the data exists, it just isn't fed into the status decision.

## Evidence (our public deployment, `analyzer.marwoj.net`)

Three random repeaters from `/api/nodes?role=repeater` snapshot, with current values:

| Node | last_seen (ADVERT) | last_relayed | relay_count_24h |
|---|---|---|---|
| FORT BEMA RPT* | ~80 min ago | **~2 min ago** | **2468** |
| SK-KE-Barca | ~80 min ago | ~5 min ago | 490 |
| WGM1 rpt | ~80 min ago | ~10 min ago | 106 |

With `infraSilentHours: 168` (7 days) all three appear correctly as Active — because the gap is shorter than 7 days. But the operator's intuition is "if I set the threshold to 2h or 24h, I want repeaters that relayed in the last 2h / 24h to count as Active, not just repeaters that adverted recently". With a 24h `infraSilentHours` and a repeater that hasn't adverted in 30h, the node falls out of Active despite relaying 1000+ packets in the last hour.

## Where the bug lives

**Frontend (`public/roles.js:355`):**

```javascript
window.getNodeStatus = function (role, lastSeenMs) {
  var isInfra = role === 'repeater' || role === 'room';
  var staleMs = isInfra ? HEALTH_THRESHOLDS.infraSilentMs : HEALTH_THRESHOLDS.nodeSilentMs;
  var age = typeof lastSeenMs === 'number' ? (Date.now() - lastSeenMs) : Infinity;
  return age < staleMs ? 'active' : 'stale';
};
```

The function takes one timestamp. Callers (`nodes.js:1146-1149` for the status filter; `nodes.js:189` for the per-node detail panel) pass `n.last_heard || n.last_seen` — `n.last_relayed` is silently dropped.

**Backend already supplies the data.** `cmd/server/routes.go:1279-1284` (in `handleNodes`) enriches each repeater/room row with:
- `last_relayed`
- `relay_active`
- `relay_count_1h`, `relay_count_24h`

So the fix is purely in the frontend — call signature plus call sites.

## Proposed fix

**Option 1 — extend `getNodeStatus()` signature:**

```javascript
window.getNodeStatus = function (role, lastSeenMs, lastRelayedMs) {
  var isInfra = role === 'repeater' || role === 'room';
  var staleMs = isInfra ? HEALTH_THRESHOLDS.infraSilentMs : HEALTH_THRESHOLDS.nodeSilentMs;
  // For infra roles, treat last_relayed as equivalent evidence of life
  var effectiveMs = isInfra && typeof lastRelayedMs === 'number'
    ? Math.max(lastSeenMs || 0, lastRelayedMs)
    : lastSeenMs;
  var age = typeof effectiveMs === 'number' && effectiveMs > 0
    ? (Date.now() - effectiveMs) : Infinity;
  return age < staleMs ? 'active' : 'stale';
};
```

Update call sites to pass `n.last_relayed`:
- `nodes.js:1144-1151` (status filter)
- `nodes.js:186-208` (per-node status info)
- Any map/live equivalents

**Option 2 — server-side merge into a single field.** Backend `handleNodes` could set `n.last_seen = max(last_seen, last_relayed)` for repeaters/rooms before returning. Simpler frontend change (single field), but harder to disambiguate "still adverting" vs "only relaying" in the UI if you ever want to show both.

I'd lean toward Option 1 — it preserves the original semantics of each field and only changes how status is *derived*.

## Severity

P2/P3 — non-fatal but misleading. Operators currently work around by setting `infraSilentHours` very high (we'll use 720h = 30d temporarily, equivalent to `retention.nodeDays`), but that makes the threshold meaningless. The right fix is to make the threshold mean "no signs of life within X" not "no ADVERT within X".

## Related

- #662 (relay activity already tracked server-side, mentioned in `RelayActiveHours` config)
- `relayActiveHours` (default 24h) is documented as "how recent a path-hop appearance must be for a repeater to be considered 'actively relaying' vs only 'alive (advert-only)'" — same intuition, just not wired into status filter

**Comment by MarekWo** [comment]
Closing as duplicate of #1598 — same root cause (status calc ignores relay activity), same fix surface. Will fold the proposal here into #1598 with additional empirical data.

**Comment by MarekWo** [comment]
Dup of #1598.
